### ISSUE TITLE
The view YtManagerApp.views.first_time.Step1ApiKeyView didn't return an HttpResponse object. It returned None instead.

### ISSUE THREAD

**Issue by saveyour** [chibicitiberiu/ytsm#60]
i am getting this error on a fresh install of ytsm. i decided to wipe my previous install from a week or so ago and try again due to some errors i was encountering and upon a full reinstall i get this error when trying to give my api key for youtube. love the app, thanks for the help!

```
Request Method: | POST
-- | --
http://ytsm/first_time/step1_apikey
2.1.4
ValueError
The view YtManagerApp.views.first_time.Step1ApiKeyView didn't return an HttpResponse object. It returned None instead.
/usr/local/lib/python3.6/dist-packages/django/core/handlers/base.py in _get_response, line 137
/usr/bin/python3
3.6.7
['/home/ubuntu/ytsm/app',  '/usr/lib/python36.zip',  '/usr/lib/python3.6',  '/usr/lib/python3.6/lib-dynload',  '/usr/local/lib/python3.6/dist-packages',  '/usr/lib/python3/dist-packages']
Sun, 6 Jan 2019 17:34:54 +0000
```

**Comment by chibicitiberiu** [comment]
Will investigate, a but busy at the moment.

**Comment by chibicitiberiu** [comment]
Fixed in #67 .
